nixos/modules/services/display-managers/sddm.nix

```diff
@@ -86,23 +86,36 @@ let
 in
 {
   imports = [
+    (mkRenamedOptionModule [ "services" "xserver" "displayManager" "sddm" "autoLogin" "minimumUid" ] [ "services" "displayManager" "sddm" "autoLogin" "minimumUid" ])
+    (mkRenamedOptionModule [ "services" "xserver" "displayManager" "sddm" "autoLogin" "relogin" ] [ "services" "displayManager" "sddm" "autoLogin" "relogin" ])
+    (mkRenamedOptionModule [ "services" "xserver" "displayManager" "sddm" "autoNumlock" ] [ "services" "displayManager" "sddm" "autoNumlock" ])
+    (mkRenamedOptionModule [ "services" "xserver" "displayManager" "sddm" "enable" ]      [ "services" "displayManager" "sddm" "enable" ])
+    (mkRenamedOptionModule [ "services" "xserver" "displayManager" "sddm" "enableHidpi" ] [ "services" "displayManager" "sddm" "enableHidpi" ])
+    (mkRenamedOptionModule [ "services" "xserver" "displayManager" "sddm" "extraPackages" ] [ "services" "displayManager" "sddm" "extraPackages" ])
+    (mkRenamedOptionModule [ "services" "xserver" "displayManager" "sddm" "package" ]     [ "services" "displayManager" "sddm" "package" ])
```

Comment:
I guess we should also update plasma6 to use the new options with the pr to avoid warnings

https://github.com/SuperSandro2000/nixpkgs/blob/sddm-wayland-only/nixos/modules/services/x11/desktop-managers/plasma6.nix#L240

Reply:
Yeah, just noticed that, too but we need to rebase the PR for that.

Reply:
I also forgot to move `services.xserver.displayManager.defaultSession`

Reply:
I did local testing without the imports and now all things should refer to the new options. I also found three more options that should be moved. The x11 module now only contains x11 bits but the scope increased again which I wanted to avoid.